test/common/testvm.py

```diff
@@ -106,6 +114,12 @@ def __init__(self, address=None, image=None, verbose=False, label=None, fetch=Tr
         self.ssh_process = None
         self.ssh_port = 22
 
+        # keep track of currently running jobs via ssh
+        # these might need to be terminated on a timeout
+        self._current_exec = []
```

Comment:
Lets just call this processes. The naming intent is that members with a _ prefix is meant to be somewhat private and only used by the class itself and derived classes.